src/status_im/contexts/wallet/send/flow_config.cljs

```diff
@@ -1,13 +1,30 @@
 (ns status-im.contexts.wallet.send.flow-config)
 
+(defn- collectible-selected?
+  [db]
+  (let [collectible-stored (-> db :wallet :ui :send :collectible)
+        tx-type            (-> db :wallet :ui :send :tx-type)]
+    (and (some? collectible-stored)
+         (= tx-type :collectible))))
+
+(defn- token-selected?
+  [db]
+  (-> db :wallet :ui :send :token some?))
+
 (def steps
```

Comment:
@ulisesmac - can you instead make a separate flow for sending an asset vs a collectible.
I think it will be cleaner this way. We should also do the same for bridging imo. 

There's also an issue with sending a collectible this way. 

The issue is this, if you are on the home page - long press on a collectible.
When you click send it will bring you to the "From" Page.
This From page shows all addresses 
- it's unclear if we want this, probably we want to show that page only if that collectible is owned by many accounts by the user.


Also in the case of a user do we really want to allow a user to select an account that doesn't have that asset to send? 🤔 
Seems like the other accounts should be greyed out, probably a separate issue.

Reply:
Yeah, since while sending a token, we also display all accounts, I wasn't sure if we really wanted to filter.

Maybe better if this filtering for sending collectibles and assets is first calrified and then solved in the different wallet areas?

Reply:
Regarding separating the flow, at the beginning it was my approach, but this infrastructure doesn't support it.

At the beginning, I was in a flow (the regular sending) and then just jumped to another flow when a collectible was picked (`wallet-flow` -> `send-collectible-flow`) and it worked for the simple sending, but once I wanted to skip some steps (particularly the collectible picking) then the flow didn't know it must jump to another flow.

So I realized that if I wanted that same screens take different flows and skip steps successfully, I should, from the beginning, stablish a flow and persist it on re-frame, and not passing the flow-id to the wizard anymore, UNLESS, you wanted to jump from one flow to another, and (IDK exactly how and when) make sure to clean the db with the flow picked, because the user could just give up, and close the flow even if it's not completed.


Since this required a significant change in the structure I thought it was not intended to do so, so I just kept our `wallet-flow` sharing all steps but skipping according to conditions based on the type of transaction.

I'm not against making big changes to the mechanism, I just was unsure, so if you think that is the best, I can do it 👍

Also, idk if we want multiple flows repeating many steps (as firsts and last steps in sending) or we want to define a flow with subflows (maybe a bit more complex to implement).

Wdyt?